### PR TITLE
Feature/homepage template grid rework

### PR DESCRIPTION
**Pull request by klepas**
Reworks the homepage mostly as per the design.

Some final tweaking left based on feedback from Gaz.

### CODE REVIEW

File: _assets/scss/design-guide.scss [hunk 1]
```diff
@@ -22,72 +22,135 @@ table {
...
 
-  .page-header__logo {
-    padding-top: 0;
-    margin-bottom: $tiny-spacing;
+  .wrapper {
+    @include ie-lte(8) {
+      @include ie-clearfix;
+    }
+
+    background-image: url('/assets/hero.svg');
```
Comment (dominikwilkowski) [inline _assets/scss/design-guide.scss]: Can we inline the SVG to save that precious http request (prior to http/2 rollout)?
E.G.:
```css
url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' width='1000' height='20' version='1.1'><polyline fill='white' points='0,0 1000,0 0,20'/></svg>") left top / 100% auto no-repeat
```

http://meyerweb.com/eric/tools/dencoder/

File: _assets/scss/design-guide.scss [hunk 2]
```diff
@@ -22,72 +22,135 @@ table {
...
+        margin-right: 5%;
+      }
+    }
+  }
+}
 
-        @include media($tablet) {
-          width: rem(64);
-        }
+.no-svg {
```
Comment (dominikwilkowski) [inline _assets/scss/design-guide.scss:97]: Just a question but should be include the `no-svg` class as a default on our html tag? That way we have a fallback when js is off?

File: _assets/scss/design-guide.scss [hunk 3]
```diff
@@ -22,72 +22,135 @@ table {
...
+// Custom styling for the inverted feedback button in the header.
+// This will likely be removed/replaced by a global site search.
+// If other guides need it we might want to look into this.
+//
+// Either way we may want to revisit how the feedback button (*cough*) is set.
+// It’s also undocumented.
+//
+// The @extend .inverted is not working in:
+// https://github.com/AusDTO/gov-au-ui-kit/blob/master/assets/sass/components/_page-header.scss#L288
+
```
Comment (dominikwilkowski) [inline _assets/scss/design-guide.scss:167]: nice comments :)

File: _assets/scss/design-guide.scss [hunk 4]
```diff
@@ -132,6 +195,52 @@ main {
...
+          @include omega(2n);
+
+          @include ie-lte(8) {
+            width: 45%;
+          }
+
+          padding-left: $medium-spacing;
+          border-left: 1px solid $border-colour;
+
+          & > h2 {
```
Comment (dominikwilkowski) [inline _assets/scss/design-guide.scss]: some deep nesting here..

File: _config.yml [hunk 5]
```diff
@@ -70,27 +70,27 @@ collections:
   getting-started:
     output: true
     title: "Getting started"
-    abstract: "TBC."
+    abstract: "Design system principles and basics for designers and developers."
     order: 0
```
Comment (dominikwilkowski) [inline _config.yml:72]: Do we actually need this in the config? I mean do we reuse it anywhere else?

**Review by dominikwilkowski — CHANGES_REQUESTED** [review]
Have a look at the svg inlining and we should be good

**Comment by dominikwilkowski** [comment]
In fact use this string for the background image @klepas 

```css
background-image: url('data:image/svg+xml;charset=US-ASCII,%3Csvg%20xmlns%3D%22http%3A//www.w3.org/2000/svg%22%20viewBox%3D%220%200%20600%20295%22%3E%3Cg%20fill%3D%22none%22%20fill-rule%3D%22evenodd%22%3E%3Cpath%20fill%3D%22%2329C1E3%22%20d%3D%22M74%2019h318v208H74z%22/%3E%3Cpath%20fill%3D%22%23244A61%22%20d%3D%22M392%20226H77V19h315v207zM234.996%206A3.002%203.002%200%200%201%20238%209a2.999%202.999%200%201%201-6%200c0-1.656%201.34-3%202.996-3zm171.616-6H63.388C61.525%200%2060%201.717%2060%203.815v236.637c0%202.098%201.525%203.815%203.388%203.815h157.614c-.015.03-.042.057-.056.088l-9.593%2021.91c-.41.94-1.47%201.985-2.353%202.32l-17.855%206.803c-.882.337-.842.612.09.612h87.527c.933%200%20.974-.275.09-.612L261%20268.586c-.883-.336-1.942-1.38-2.355-2.322l-9.595-21.91c-.013-.03-.04-.056-.054-.087h157.616c1.863%200%203.388-1.717%203.388-3.815V3.815C410%201.717%20408.475%200%20406.612%200zM60%200h136%22/%3E%3Cpath%20fill%3D%22%234597B9%22%20d%3D%22M85%2026h301v30H85z%22/%3E%3Cpath%20fill%3D%22%2397E4F2%22%20d%3D%22M85%2062h41v163H85zm50%2088h252v76H135zm0-47h79v41h-79zm85%200h79v41h-79zm86%200h79v41h-79zM144.133%2077.854h7.168v9.367c-1.16.38-2.255.646-3.28.8-1.026.15-2.075.227-3.146.227-2.727%200-4.81-.8-6.247-2.404-1.438-1.602-2.156-3.903-2.156-6.902%200-2.915.835-5.19%202.503-6.82%201.67-1.632%203.982-2.447%206.94-2.447%201.853%200%203.64.37%205.363%201.112l-1.273%203.065a9.08%209.08%200%200%200-4.115-.99c-1.656%200-2.983.557-3.98%201.67-.997%201.112-1.495%202.607-1.495%204.485%200%201.96.402%203.46%201.205%204.493.803%201.034%201.97%201.55%203.503%201.55.8%200%201.612-.082%202.436-.246v-3.77h-3.423v-3.188zm14.212-7.922l.173.272c-.412%201.623-1.137%203.596-2.175%205.92h-2.707c.57-2.44.98-4.503%201.236-6.192h3.473zm6.822%2018.315c-1.623%200-2.898-.63-3.825-1.89-.927-1.26-1.39-3.008-1.39-5.24%200-2.266.47-4.03%201.414-5.296.944-1.264%202.244-1.896%203.9-1.896%201.738%200%203.064.675%203.98%202.027h.123c-.19-1.03-.286-1.948-.286-2.755V68.77h3.782V88h-2.892l-.73-1.792h-.16c-.857%201.36-2.162%202.04-3.917%202.04zm1.322-3.003c.963%200%201.67-.28%202.118-.84.45-.56.695-1.512.736-2.855v-.41c0-1.482-.23-2.545-.686-3.187-.457-.643-1.2-.964-2.23-.964-.84%200-1.494.355-1.96%201.068-.465.712-.698%201.748-.698%203.108s.235%202.38.705%203.06c.47.678%201.14%201.018%202.014%201.018zM185.582%2088l-.73-1.878h-.098c-.635.8-1.288%201.353-1.96%201.662-.67.31-1.546.463-2.625.463-1.327%200-2.37-.38-3.133-1.137-.762-.758-1.143-1.837-1.143-3.238%200-1.466.513-2.548%201.538-3.244%201.026-.696%202.573-1.08%204.64-1.155l2.398-.074v-.607c0-1.4-.716-2.1-2.15-2.1-1.104%200-2.402.333-3.893%201l-1.248-2.546c1.59-.832%203.352-1.248%205.288-1.248%201.854%200%203.275.403%204.264%201.21.99.808%201.483%202.035%201.483%203.683V88h-2.632zm-1.113-6.402l-1.458.05c-1.096.033-1.91.23-2.447.593-.535.364-.803.916-.803%201.657%200%201.063.61%201.594%201.83%201.594.872%200%201.57-.25%202.094-.753.523-.502.784-1.17.784-2.002V81.6zm5.648-7.415h4.128l2.607%207.774c.223.675.375%201.475.458%202.397h.075c.09-.848.268-1.647.53-2.397l2.56-7.774h4.04l-5.845%2015.584c-.535%201.442-1.3%202.52-2.292%203.238-.993.717-2.153%201.075-3.48%201.075-.65%200-1.29-.07-1.915-.21v-2.99c.453.107.948.16%201.483.16.668%200%201.25-.204%201.75-.612.497-.407.887-1.023%201.167-1.847l.222-.68-5.487-13.717zm20.07%2011.16c-.428%201.664-1.153%203.637-2.175%205.92h-2.72c.537-2.192.953-4.26%201.25-6.204h3.46l.185.283zM231.766%2088l-1.063-4.832-1.434-6.105h-.087L226.663%2088h-4.055l-3.93-13.817h3.757l1.594%206.118c.255%201.097.514%202.608.778%204.536h.074c.033-.626.177-1.62.433-2.978l.197-1.05%201.706-6.625h4.152l1.62%206.624c.032.182.084.45.154.804.07.355.138.732.204%201.132.065.4.125.79.18%201.174.052.383.083.69.09.92h.076c.074-.593.206-1.407.395-2.44.19-1.034.325-1.733.407-2.095l1.656-6.117h3.695L235.868%2088h-4.102zm13.31-6.933c0%201.368.224%202.402.673%203.102.448.7%201.18%201.05%202.193%201.05%201.005%200%201.728-.35%202.17-1.045.44-.696.66-1.732.66-3.108%200-1.368-.223-2.393-.668-3.077-.444-.684-1.174-1.026-2.187-1.026-1.005%200-1.73.34-2.175%201.02-.445.68-.667%201.707-.667%203.083zm9.553%200c0%202.25-.595%204.008-1.78%205.277-1.187%201.27-2.84%201.903-4.957%201.903-1.326%200-2.496-.29-3.51-.87-1.013-.582-1.79-1.416-2.335-2.504-.544-1.087-.816-2.356-.816-3.806%200-2.257.59-4.012%201.767-5.265%201.178-1.252%202.834-1.878%204.967-1.878%201.327%200%202.497.288%203.51.865%201.014.576%201.792%201.404%202.336%202.483.544%201.08.816%202.344.816%203.794zm10.825-7.143c.51%200%20.935.037%201.272.11l-.284%203.536a4.287%204.287%200%200%200-1.112-.124c-1.202%200-2.14.31-2.81.927-.672.618-1.008%201.483-1.008%202.595V88h-3.77V74.183h2.856l.556%202.324h.185a5.174%205.174%200%200%201%201.736-1.873%204.29%204.29%200%200%201%202.38-.71zM273.005%2088h-3.77V68.77h3.77V88zm8.33.247c-1.623%200-2.898-.63-3.825-1.89-.927-1.26-1.39-3.008-1.39-5.24%200-2.266.47-4.03%201.415-5.296.943-1.264%202.243-1.896%203.9-1.896%201.737%200%203.064.675%203.978%202.027h.124c-.19-1.03-.284-1.948-.284-2.755V68.77h3.78V88h-2.89l-.73-1.792h-.16c-.857%201.36-2.163%202.04-3.918%202.04zm1.322-3.003c.964%200%201.67-.28%202.12-.84.45-.56.694-1.512.735-2.855v-.41c0-1.482-.23-2.545-.686-3.187-.457-.643-1.2-.964-2.23-.964-.84%200-1.494.355-1.96%201.068-.465.712-.698%201.748-.698%203.108s.235%202.38.705%203.06c.47.678%201.14%201.018%202.014%201.018zm9.788.99c0-.693.185-1.216.556-1.57.372-.355.912-.532%201.62-.532.684%200%201.213.18%201.588.544.375.362.563.88.563%201.557%200%20.65-.19%201.163-.568%201.538-.38.376-.906.564-1.582.564-.692%200-1.227-.184-1.606-.55-.38-.367-.57-.884-.57-1.55zM306%2065h79v4h-79zm0%207h79v4h-79zm0%207h79v4h-79zm0%207h79v4h-79zm0%207h53v4h-53z%22/%3E%3Cpath%20fill%3D%22%23244A61%22%20d%3D%22M468%20278h126.574c1.884%200%203.426%201.723%203.426%203.828v7.344c0%202.105-1.542%203.828-3.426%203.828H288.426c-1.884%200-3.426-1.723-3.426-3.828v-7.344c0-2.105%201.542-3.828%203.426-3.828H416v4.055c0%201.07.772%201.945%201.715%201.945h48.57c.943%200%201.715-.875%201.715-1.945V278zm86-17.948c0%201.07-.77%201.948-1.71%201.948H329.71c-.94%200-1.71-.877-1.71-1.948V114.948c0-1.07.77-1.948%201.71-1.948h222.58c.94%200%201.71.877%201.71%201.948v145.104zM441.497%20106c.826%200%201.503.67%201.503%201.503%200%20.828-.677%201.497-1.503%201.497-.828%200-1.497-.67-1.497-1.497%200-.832.67-1.503%201.497-1.503zm-122.787-4c-.94%200-1.71.878-1.71%201.95v168.1c0%201.072.77%201.95%201.71%201.95h244.58c.94%200%201.71-.878%201.71-1.95v-168.1c0-1.072-.77-1.95-1.71-1.95H318.71z%22/%3E%3Cpath%20fill%3D%22%2329C1E3%22%20d%3D%22M328%20113h227v151H328z%22/%3E%3Cpath%20fill%3D%22%2397E4F2%22%20d%3D%22M349.103%20143.11h5.575v7.284c-.903.295-1.754.502-2.552.62-.798.12-1.613.178-2.446.178-2.12%200-3.74-.623-4.86-1.87-1.117-1.246-1.676-3.035-1.676-5.368%200-2.268.65-4.037%201.946-5.305%201.298-1.27%203.097-1.905%205.397-1.905%201.442%200%202.833.29%204.172.865l-.99%202.384a7.086%207.086%200%200%200-3.202-.77c-1.288%200-2.32.434-3.095%201.3-.775.864-1.163%202.027-1.163%203.488%200%201.525.312%202.69.937%203.494.625.804%201.533%201.206%202.725%201.206.622%200%201.253-.064%201.894-.192v-2.932h-2.663v-2.48zm11.054-6.163l.135.212c-.32%201.26-.885%202.796-1.692%204.603h-2.105c.442-1.897.763-3.502.96-4.816h2.702zm5.306%2014.245c-1.262%200-2.254-.49-2.975-1.47-.72-.98-1.08-2.34-1.08-4.076%200-1.762.366-3.135%201.1-4.12.733-.982%201.744-1.474%203.032-1.474%201.352%200%202.384.525%203.095%201.576h.096a12.09%2012.09%200%200%201-.22-2.143v-3.44h2.94V151h-2.248l-.567-1.394h-.125c-.666%201.058-1.682%201.586-3.047%201.586zm1.028-2.335c.75%200%201.3-.218%201.65-.654.35-.436.54-1.176.572-2.22v-.318c0-1.153-.178-1.98-.534-2.48-.355-.5-.934-.75-1.735-.75-.653%200-1.16.278-1.523.832-.362.554-.543%201.36-.543%202.418%200%201.057.182%201.85.548%202.38.365.527.887.792%201.566.792zM381.343%20151l-.567-1.46h-.077c-.493.62-1%201.05-1.523%201.292-.523.24-1.204.36-2.043.36-1.032%200-1.844-.295-2.437-.884-.592-.59-.89-1.43-.89-2.518%200-1.14.4-1.982%201.198-2.524.798-.54%202-.84%203.61-.898l1.864-.058v-.47c0-1.09-.558-1.635-1.673-1.635-.858%200-1.867.26-3.027.78l-.97-1.982c1.236-.647%202.607-.97%204.113-.97%201.442%200%202.547.314%203.316.942.77.628%201.153%201.582%201.153%202.864V151h-2.048zm-.865-4.98l-1.134.04c-.853.025-1.487.18-1.903.46-.417.283-.625.712-.625%201.29%200%20.825.474%201.24%201.422%201.24.68%200%201.223-.197%201.63-.588.407-.39.61-.91.61-1.557v-.884zm4.393-5.766h3.21l2.028%206.046c.173.525.292%201.147.356%201.864h.058c.07-.66.208-1.28.413-1.864l1.99-6.046h3.143l-4.547%2012.12c-.415%201.122-1.01%201.96-1.782%202.52-.772.556-1.674.835-2.706.835a6.81%206.81%200%200%201-1.49-.164v-2.326c.353.083.737.125%201.154.125.52%200%20.972-.16%201.36-.476.388-.318.69-.797.908-1.438l.173-.53-4.267-10.668zm15.61%208.68c-.334%201.294-.898%202.83-1.692%204.604h-2.115c.417-1.705.74-3.313.97-4.826h2.692l.145.22zM417.262%20151l-.826-3.758-1.115-4.75h-.067l-1.96%208.508h-3.153l-3.057-10.746h2.922l1.24%204.758c.2.852.4%202.028.606%203.527h.058c.026-.488.138-1.26.337-2.317l.154-.817%201.325-5.152h3.23l1.26%205.152c.024.14.065.35.12.625a38.076%2038.076%200%200%201%20.298%201.793c.04.298.065.537.07.716h.06c.056-.462.16-1.095.306-1.9a48.59%2048.59%200%200%201%20.316-1.628l1.288-4.758h2.874L420.453%20151h-3.19zm10.352-5.392c0%201.063.175%201.868.524%202.412.35.545.918.817%201.706.817.782%200%201.344-.27%201.687-.812.344-.54.516-1.347.516-2.417%200-1.064-.173-1.862-.52-2.394-.346-.532-.913-.798-1.7-.798-.783%200-1.347.265-1.693.793-.346.528-.52%201.328-.52%202.398zm7.43%200c0%201.75-.46%203.117-1.384%204.104-.922.987-2.207%201.48-3.854%201.48-1.032%200-1.942-.226-2.73-.677a4.508%204.508%200%200%201-1.817-1.947c-.423-.846-.635-1.833-.635-2.96%200-1.756.458-3.12%201.375-4.095.916-.974%202.204-1.46%203.864-1.46%201.03%200%201.94.223%202.73.672a4.485%204.485%200%200%201%201.816%201.932c.423.84.634%201.823.634%202.95zm8.42-5.556c.398%200%20.728.03.99.086l-.22%202.75a3.322%203.322%200%200%200-.865-.097c-.937%200-1.666.242-2.188.722-.522.48-.784%201.154-.784%202.02V151h-2.93v-10.746h2.22l.432%201.807h.144a4.033%204.033%200%200%201%201.35-1.455%203.318%203.318%200%200%201%201.85-.553zM449.34%20151h-2.932v-14.956h2.932V151zm6.478.192c-1.262%200-2.254-.49-2.975-1.47-.72-.98-1.08-2.34-1.08-4.076%200-1.762.367-3.135%201.1-4.12.734-.982%201.745-1.474%203.033-1.474%201.352%200%202.384.525%203.095%201.576h.096c-.147-.8-.22-1.515-.22-2.143v-3.44h2.94V151h-2.25l-.566-1.394h-.125c-.666%201.058-1.682%201.586-3.047%201.586zm1.03-2.335c.748%200%201.298-.218%201.647-.654.35-.436.54-1.176.572-2.22v-.318c0-1.153-.178-1.98-.534-2.48-.354-.5-.932-.75-1.733-.75-.654%200-1.162.278-1.524.832-.362.554-.543%201.36-.543%202.418%200%201.057.183%201.85.548%202.38.365.527.887.792%201.567.792zm7.61.768c0-.538.145-.945.434-1.22.288-.276.708-.414%201.26-.414.53%200%20.943.142%201.234.424.292.282.438.686.438%201.21%200%20.508-.148.907-.443%201.198-.295.292-.706.438-1.23.438-.54%200-.956-.143-1.25-.428-.295-.285-.443-.687-.443-1.207zM478%20134h62v4h-62zm0%207h62v4h-62zm0%207h62v4h-62zm0%207h41v4h-41zm-136%2044h200v57H342zm0-35h63v31h-63zm137%200h63v31h-63zm-69%200h63v31h-63z%22/%3E%3Cpath%20fill%3D%22%23244A61%22%20d%3D%22M104%20268H8V141h96v127zm-47%2018c-3.312%200-6-2.69-6-6s2.688-6%206-6c3.32%200%206%202.69%206%206s-2.68%206-6%206zm-.5-158c1.384%200%202.5%201.125%202.5%202.505A2.5%202.5%200%201%201%2056.5%20128zm52.18-8H3.32C1.493%20120%200%20121.68%200%20123.732v164.536C0%20290.32%201.494%20292%203.32%20292h105.36c1.826%200%203.32-1.68%203.32-3.732V123.732c0-2.053-1.494-3.732-3.32-3.732z%22/%3E%3Cpath%20fill%3D%22%2329C1E3%22%20d%3D%22M7%20138h97v131H7z%22/%3E%3Cpath%20fill%3D%22%2397E4F2%22%20d%3D%22M13%20182h79v4H13zm0%207h79v4H13zm0%207h79v4H13zm0%207h79v4H13zm0%207h53v4H13zm0%2011h27v21H13zm0%2024h27v21H13zm30-24h27v21H43zm0%2024h27v21H43zm30-24h27v21H73zm0%2024h27v21H73zm-55.434-83.073h3.584v4.684c-.58.19-1.127.323-1.64.4-.513.075-1.037.114-1.573.114-1.363%200-2.405-.4-3.123-1.202-.72-.802-1.08-1.952-1.08-3.45%200-1.46.418-2.597%201.253-3.412.834-.816%201.99-1.224%203.47-1.224.926%200%201.82.186%202.68.556l-.635%201.533a4.543%204.543%200%200%200-2.058-.494c-.828%200-1.49.28-1.99.835-.498.556-.748%201.304-.748%202.243%200%20.98.2%201.73.603%202.246.4.517.985.775%201.75.775.4%200%20.806-.042%201.218-.124v-1.885h-1.712v-1.593zm7.106-3.96l.087.135c-.207.812-.57%201.798-1.09%202.96H22.32c.284-1.22.49-2.25.618-3.096h1.736zm3.41%209.157c-.81%200-1.448-.316-1.91-.946-.465-.63-.696-1.503-.696-2.62%200-1.133.236-2.015.707-2.648.472-.632%201.122-.948%201.95-.948.87%200%201.532.338%201.99%201.013h.06a7.848%207.848%200%200%201-.14-1.378v-2.212h1.89V167h-1.446l-.365-.896h-.08c-.428.68-1.08%201.02-1.96%201.02zm.663-1.502c.48%200%20.835-.14%201.06-.42.224-.28.346-.756.367-1.428v-.203c0-.74-.114-1.272-.343-1.594-.23-.32-.6-.482-1.116-.482-.42%200-.747.18-.98.535-.233.355-.35.873-.35%201.553%200%20.68.118%201.19.353%201.53.235.34.57.51%201.008.51zM38.29%20167l-.363-.94h-.05c-.317.4-.643.677-.98.832-.335.154-.773.232-1.312.232-.663%200-1.186-.19-1.567-.57-.38-.378-.57-.918-.57-1.618%200-.733.255-1.274.768-1.622.513-.348%201.286-.54%202.32-.578l1.2-.037v-.303c0-.7-.36-1.05-1.076-1.05-.552%200-1.2.166-1.946.5l-.625-1.273c.795-.416%201.676-.624%202.644-.624.927%200%201.638.2%202.132.605.494.404.74%201.018.74%201.842V167H38.29zm-.555-3.2l-.73.024c-.547.016-.955.115-1.222.297-.268.182-.402.458-.402.83%200%20.53.306.796.915.796.437%200%20.786-.126%201.048-.377.26-.253.392-.586.392-1.002v-.57zm2.824-3.708h2.063l1.304%203.886c.11.338.187.738.228%201.2h.037c.046-.425.134-.825.266-1.2l1.28-3.886h2.02l-2.923%207.792c-.268.72-.65%201.26-1.146%201.62-.498.357-1.077.536-1.74.536-.326%200-.645-.035-.958-.105v-1.495c.226.053.473.08.74.08.335%200%20.626-.102.875-.306.25-.204.444-.512.584-.924l.113-.34-2.744-6.858zm10.034%205.58c-.214.832-.577%201.818-1.087%202.96h-1.36a35.73%2035.73%200%200%200%20.624-3.103h1.73l.094.14zM61.384%20167l-.533-2.416-.715-3.053h-.044L58.83%20167h-2.026l-1.965-6.908h1.878l.797%203.058c.127.548.257%201.304.39%202.268h.036c.018-.313.09-.81.217-1.49l.1-.524.852-3.312h2.075l.81%203.312c.016.09.042.224.077.4a24.984%2024.984%200%200%201%20.192%201.155c.026.19.042.345.046.46h.037c.037-.296.103-.703.198-1.22.094-.517.162-.866.203-1.048l.828-3.058h1.848L63.434%20167h-2.05zm6.654-3.467c0%20.684.112%201.2.337%201.55.224.352.59.527%201.096.527.504%200%20.865-.174%201.086-.522.22-.35.33-.867.33-1.555%200-.683-.11-1.196-.333-1.538-.223-.342-.587-.513-1.094-.513-.504%200-.866.17-1.09.51-.22.34-.332.854-.332%201.54zm4.776%200c0%201.125-.296%202.005-.89%202.64-.593.633-1.42.95-2.477.95-.664%200-1.25-.145-1.755-.435a2.902%202.902%200%200%201-1.168-1.25c-.272-.545-.408-1.18-.408-1.905%200-1.128.295-2.006.884-2.632.59-.625%201.417-.938%202.484-.938.663%200%201.248.144%201.755.432a2.9%202.9%200%200%201%201.166%201.242c.272.54.408%201.172.408%201.897zm5.413-3.57c.256%200%20.468.017.637.055l-.142%201.767a2.219%202.219%200%200%200-.557-.062c-.6%200-1.07.154-1.405.463-.336.31-.504.742-.504%201.298V167H74.37v-6.908h1.43l.277%201.16h.093c.214-.386.503-.698.868-.935.364-.237.76-.355%201.19-.355zM82.003%20167h-1.885v-9.615h1.885V167zm4.164.124c-.81%200-1.45-.316-1.912-.946-.463-.63-.695-1.503-.695-2.62%200-1.133.236-2.015.707-2.648.472-.632%201.122-.948%201.95-.948.87%200%201.532.338%201.99%201.013h.06a7.848%207.848%200%200%201-.14-1.378v-2.212h1.89V167H88.57l-.363-.896h-.08c-.43.68-1.082%201.02-1.96%201.02zm.662-1.502c.48%200%20.834-.14%201.058-.42.225-.28.347-.756.368-1.428v-.203c0-.74-.114-1.272-.343-1.594-.23-.32-.6-.482-1.115-.482-.42%200-.747.18-.98.535-.232.355-.35.873-.35%201.553%200%20.68.12%201.19.353%201.53.236.34.572.51%201.01.51zm4.893.494c0-.346.092-.607.278-.784.186-.178.456-.266.81-.266.342%200%20.607.09.794.272.188.18.28.44.28.778%200%20.326-.093.582-.283.77-.188.187-.452.28-.79.28-.346%200-.614-.09-.803-.274-.19-.183-.284-.442-.284-.776z%22/%3E%3Cpath%20fill%3D%22%23244A61%22%20d%3D%22M175%20274h-49v-90h49v90zm-24%2015c-2.762%200-5-2.243-5-5%200-2.764%202.238-5%205-5a5%205%200%201%201%200%2010zm-7-109h12v1.366h-12V180zm6.5-5a1.5%201.5%200%201%201-.001%203.001A1.5%201.5%200%200%201%20150.5%20175zm25.374-4h-50.748c-1.72%200-3.126%201.55-3.126%203.443v115.114c0%201.893%201.407%203.443%203.126%203.443h50.748c1.72%200%203.126-1.55%203.126-3.443V174.443c0-1.893-1.407-3.443-3.126-3.443z%22/%3E%3Cpath%20fill%3D%22%2329C1E3%22%20d%3D%22M125%20184h51v91h-51z%22/%3E%3Cpath%20fill%3D%22%2397E4F2%22%20d%3D%22M130%20241h41v4h-41zm0-7h41v4h-41zm0%2014h41v4h-41zm0%207h41v4h-41zm0%207h41v4h-41zm0%207h27v4h-27zm3.566-68.073h3.584v4.684c-.58.19-1.127.323-1.64.4-.513.075-1.037.114-1.573.114-1.363%200-2.405-.4-3.123-1.202-.72-.802-1.08-1.952-1.08-3.45%200-1.46.418-2.597%201.253-3.412.834-.816%201.99-1.224%203.47-1.224.926%200%201.82.186%202.68.556l-.635%201.533a4.543%204.543%200%200%200-2.058-.494c-.828%200-1.49.28-1.99.835-.498.556-.748%201.304-.748%202.243%200%20.98.2%201.73.603%202.246.4.517.985.775%201.75.775.4%200%20.806-.042%201.218-.124v-1.885h-1.712v-1.593zm7.106-3.96l.087.135c-.207.812-.57%201.798-1.09%202.96h-1.352c.284-1.22.49-2.25.618-3.096h1.736zm3.41%209.157c-.81%200-1.448-.316-1.91-.946-.465-.63-.696-1.503-.696-2.62%200-1.133.236-2.015.707-2.648.472-.632%201.122-.948%201.95-.948.87%200%201.532.338%201.99%201.013h.06a7.848%207.848%200%200%201-.14-1.378v-2.212h1.89V206h-1.446l-.365-.896h-.08c-.428.68-1.08%201.02-1.96%201.02zm.663-1.502c.48%200%20.835-.14%201.06-.42.224-.28.346-.756.367-1.428v-.203c0-.74-.114-1.272-.343-1.594-.23-.32-.6-.482-1.116-.482-.42%200-.747.18-.98.535-.233.355-.35.873-.35%201.553%200%20.68.118%201.19.353%201.53.235.34.57.51%201.008.51zM154.29%20206l-.363-.94h-.05c-.317.4-.643.677-.98.832-.335.154-.773.232-1.312.232-.663%200-1.186-.19-1.567-.57-.38-.378-.57-.918-.57-1.618%200-.733.255-1.274.768-1.622.513-.348%201.286-.54%202.32-.578l1.2-.037v-.303c0-.7-.36-1.05-1.076-1.05-.552%200-1.2.166-1.946.5l-.625-1.273c.795-.416%201.676-.624%202.644-.624.927%200%201.638.2%202.132.605.494.404.74%201.018.74%201.842V206h-1.315zm-.555-3.2l-.73.024c-.547.016-.955.115-1.222.297-.268.182-.402.458-.402.83%200%20.53.306.796.915.796.437%200%20.786-.126%201.048-.377.26-.253.392-.586.392-1.002v-.57zm2.824-3.708h2.063l1.304%203.886c.11.338.187.738.228%201.2h.037c.046-.425.134-.825.266-1.2l1.28-3.886h2.02l-2.923%207.792c-.268.72-.65%201.26-1.146%201.62-.498.357-1.077.536-1.74.536-.326%200-.645-.035-.958-.105v-1.495c.226.053.473.08.74.08.335%200%20.626-.102.875-.306.25-.204.444-.512.584-.924l.113-.34-2.744-6.858zm10.034%205.58c-.214.832-.577%201.818-1.087%202.96h-1.36a35.73%2035.73%200%200%200%20.624-3.103h1.73l.094.14zM135.667%20217l-.53-2.416-.718-3.053h-.044l-1.26%205.47h-2.027l-1.966-6.908H131l.797%203.058c.127.548.257%201.304.388%202.268h.038c.016-.313.088-.81.216-1.49l.1-.524.85-3.312h2.078l.81%203.312c.016.09.04.224.076.4a24.984%2024.984%200%200%201%20.192%201.155c.027.19.042.345.046.46h.037c.038-.296.104-.703.2-1.22.094-.517.162-.866.203-1.048l.828-3.058h1.848l-1.99%206.908h-2.05zm6.655-3.467c0%20.684.113%201.2.337%201.55.224.352.59.527%201.096.527.502%200%20.864-.174%201.084-.522.22-.35.33-.867.33-1.555%200-.683-.11-1.196-.333-1.538-.222-.342-.587-.513-1.094-.513-.502%200-.865.17-1.087.51-.222.34-.334.854-.334%201.54zm4.777%200c0%201.125-.298%202.005-.89%202.64-.594.633-1.42.95-2.48.95-.662%200-1.247-.145-1.754-.435a2.908%202.908%200%200%201-1.168-1.25c-.272-.545-.407-1.18-.407-1.905%200-1.128.295-2.006.884-2.632.59-.625%201.417-.938%202.484-.938.663%200%201.248.144%201.755.432.507.29.896.703%201.168%201.242.273.54.41%201.172.41%201.897zm5.412-3.57c.255%200%20.467.017.636.055l-.142%201.767a2.21%202.21%200%200%200-.556-.062c-.602%200-1.07.154-1.406.463-.336.31-.503.742-.503%201.298V217h-1.884v-6.908h1.427l.278%201.16h.094c.214-.386.504-.698.868-.935.365-.237.76-.355%201.19-.355zm3.775%207.037h-1.884v-9.615h1.884V217zm4.165.124c-.812%200-1.45-.316-1.913-.946-.464-.63-.696-1.503-.696-2.62%200-1.133.236-2.015.708-2.648.472-.632%201.12-.948%201.95-.948.87%200%201.532.338%201.99%201.013h.06a7.753%207.753%200%200%201-.14-1.378v-2.212h1.89V217h-1.446l-.365-.896h-.08c-.428.68-1.08%201.02-1.958%201.02zm.66-1.502c.483%200%20.836-.14%201.06-.42.225-.28.348-.756.37-1.428v-.203c0-.74-.116-1.272-.344-1.594-.23-.32-.6-.482-1.116-.482-.42%200-.746.18-.98.535-.232.355-.348.873-.348%201.553%200%20.68.117%201.19.352%201.53.235.34.57.51%201.007.51zm4.895.494c0-.346.093-.607.278-.784.185-.178.455-.266.81-.266.34%200%20.606.09.794.272.186.18.28.44.28.778%200%20.326-.095.582-.285.77-.19.187-.453.28-.79.28-.346%200-.614-.09-.804-.274-.188-.183-.283-.442-.283-.776z%22/%3E%3C/g%3E%3C/svg%3E');
```

File: _assets/scss/design-guide.scss (hunk 2 above)
Comment (klepas) [reply _assets/scss/design-guide.scss:97]: Good thinking… should I add it directly to the `<html>`?

Comment (dominikwilkowski) [reply _assets/scss/design-guide.scss:97]: I would

Comment (klepas) [reply _assets/scss/design-guide.scss:97]: Noting this is resolved b/c modernizr’s behaviour has changed slightly.

**Review by dominikwilkowski — APPROVED** [review]

**Pull request merged**